### ISSUE TITLE
Server di Videoconferenze

### ISSUE THREAD

**Issue by ehack-italy** [emergenzeHack/covid19italia_segnalazioni#1136]
<pre><yamldata>
Da_chi_offerta: autistici.org
Descrizione: 'Viste le circostanze abbiamo deciso di rendere disponibile, su base
  sperimentale e per un periodo limitato di tempo, un servizio di videoconferenza
  aperto a tutti, per supportare attività di gruppo a distanza.

  Il servizio è disponibile su vc.autistici.org ed è stato implementato usando il
  software open source Jitsi.

  Il servizio è fruibile liberamente da un computer con webcam o da uno smartphone
  (usando l’app di Jitsi), senza bisogno di effettuare alcuna registrazione o autenticazione
  (ovvero è possibile usarlo anche se non si è già utenti di autistici.org). La capacità
  è limitata, e nel caso dovesse terminare, le condizioni di accesso potrebbero cambiare
  oppure espanderemo la capacità, vedremo.

  Ci interessa però parlarvi di alcune questioni tecniche che è importante sapere
  prima di usare questo strumento.

  Autistici/Inventati non mantiene alcuna registrazione della chiamata e nessun identificativo
  personale dei partecipanti. Questo ovviamente non vale per le persone che partecipano
  alla chiamata con voi, perciò se non volete che sconosciuti partecipino alla vostra
  chiamata vi consigliamo di impostare una password per la stanza.

  Il servizio è pensato per gruppi che condividono la nostra policy, in caso di abuso
  non ci faremo problemi ad intervenire

  Pare ci siano incompatibilità con versioni di Firefox non sufficientemente recenti,
  quindi in caso riscontraste problemi vi consigliamo prima di provare con Chromium.

  In alcuni casi Jitsi effettua connessioni peer-to-peer, che implica che il proprio
  indirizzo IP può essere rivelato agli altri partecipanti.

  Ricordate che è impossibile impedire agli altri partecipanti di registrare le conversazioni,
  e che le videoconferenze sono suscettibili alle intercettazioni elettroniche ED
  ambientali contemporaneamente.

  Detto questo, non ci resta che augurarvi buon uso del nostro nuovo servizio.

  In caso riscontraste problemi ricordate che potete segnalarli all’indirizzo email
  help@autistici.org.

  Buona organizzazione!'
Destinatari: privati, lavoratori, piccole realtà
Link: https://vc.autistici.org/
Natura: didattica
Titolo: Server di Videoconferenze
</yamldata></pre>

**Comment by luciaroma** [comment]
@chiccap @cristigalas io non la accetterei perchè si tratta di un servizio di videoconferenza semplicemente alternativo ai vari meet, skype etc... non ha destinatari specifici (il sito potrebbe trarre in inganno). Che ne pensate?

**Comment by cristigalas** [comment]
No, no accettiamo. È un servizio di video conferenze che usa software open source, come questo qui #731
